MANIFEST.in

```diff
@@ -2,4 +2,7 @@ include versioneer.py
 include pyogrio/_version.py
 include pyogrio/*.pyx pyogrio/*.pxd
 exclude pyogrio/*.c
+include LICENSE
+include pyproject.toml
+exclude MANIFEST.in
```

Comment:
I am going to change this back (I think it is a left-over from some experimentation to get the tests working?), as my understanding is that this should not influence wheels but only sdist. This might still be a valid change for the sdist (didn't check that here), but then let's do that in a separate PR focused on sdist.

Reply:
That's fine; I added that while trying to get the pytest tests working, based on adapting what we have in `pygeos`.  Certainly fine to deal with it in later PR.